README.md

```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
+
+- Support multiple architectures
 
-## What do you mean by "Official"?
+- Exemplify Dockerfile best practices
 
-In many cases, the images in this program are not only supported but also *maintained* directly by the relevant upstream projects.
+- Actively rebuilt for updates and security fixes
 
-For some, they're developed in collaboration with the upstream project (or with the explicit blessing of the upstream project).
+- Adhere to upstream recommendations
 
-In all cases, we strive to create images that are true to the upstream project's vision for how their project is intended to be consumed, occasionally adding additional behavior to make them more friendly to use within Docker / containerization in general.
+- Add minimal quality-of-life behavior for the container environment where fit
+
+See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
 
-The name of this program was chosen in an attempt to reflect that upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry).
```

Comment:
I think this blurb should probably move into our FAQ somewhere (since the name definitely generates questions on the regular).

Reply:
https://github.com/docker-library/faq/pull/19